protos/piecestore/piece_store.proto

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+
+package piecestoreroutes;
+
+service PieceStoreRoutes {
+  rpc Piece(PieceHash) returns (PieceSummary) {}
+
+  rpc Retrieve(PieceRetrieval) returns (stream PieceRetrievalStream) {}
+
+  rpc Store(stream PieceStore) returns (PieceStoreSummary) {}
+
+  rpc Delete(PieceDelete) returns (PieceDeleteSummary) {}
+
+}
+
+message PieceStore {
+  string hash = 1;
```

Comment:
maybe we should just call this the name or the id or something since it isn't necessarily the hash of the data